scalatest/src/main/scala/org/scalatest/events/Event.scala

```diff
@@ -250,8 +250,10 @@ sealed abstract class Event extends Ordered[Event] with Product with Serializabl
       }
     }
 
-    def stackTrace(st: StackTraceElement): String =
+    def stackTrace(_st: StackTraceElement | Null): String = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.